specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/2016-08-10/service.json

```diff
@@ -6086,6 +6224,13 @@
             "$ref": "#/definitions/HealthError"
           }
         },
+        "failoverHealthErrorDetails": {
```

Comment:
So you the health status property has been added to the GETs of fabric etc. Why do you need a separate API to just get the healthStatus? Also, new properties have been added all across this swagger. This would require an api-version change on both service and swagger.